src/app/multi-editor/multi-editor.component.ts

```diff
@@ -175,10 +180,9 @@ export class MultiEditorComponent implements OnInit {
       })
       .catch(error => {
         if (error.json().message) {
```

Comment:
`Response.json()` is actually parsing response body each time is called
https://github.com/angular/angular/blob/5.2.0/packages/http/src/body.ts#L26 you can see here.

So what you should do is
```typescript
const errorMessage = error.json().message;
if (errorMessage) {
    this.toastr.error(errorMessage);
}
```